<launch>
<node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
  <param name="robot_base_frame" value="base_link"/>
  <param name="costmap_topic" value="map"/>
  <param name="costmap_updates_topic" value="map_updates"/>
  <param name="visualize" value="true"/>
  <param name="planner_frequency" value="1.0"/>   <!-- 边界及nbv搜索的频率 -->
  <param name="progress_timeout" value="30.0"/><!-- 如果一个goal超过progress_timeout秒，仍无法到达），则加入将goal黑名单。 -->
  <param name="potential_scale" value="3.0"/>
  <param name="orientation_scale" value="0.0"/>
  <param name="gain_scale" value="1.0"/> <!-- 以上3个变量，是frontier cost的评价函数的系数 -->
  <param name="transform_tolerance" value="0.3"/>
  <param name="min_frontier_size" value="0.75"/>
  <param name="only_viusalize_frontier" value="false"/><!-- 是否同时可视化 frontier和robot？ 原本是true。我认为应该改为false，才能-->
</node>
</launch>